ilc/server/errorHandler/ErrorHandler.js

```diff
@@ -80,11 +83,34 @@ module.exports = class ErrorHandler {
             });
 
             this.#logger.error(e);
+            this.#writeStaticError(nres);
+        }
+    }
 
-            nres.statusCode = 500;
-            nres.write('Oops! Something went wrong. Pls try to refresh page or contact support.');
-            nres.end();
+    #writeStaticError(nres) {
+        nres.statusCode = 500;
+        let content = this.#readStaticErrorPage(defaultErrorPage);
+
+        nres.write(content);
+        nres.end();
+    }
+
+    #readStaticErrorPage(defaultContent) {
+        if (this.#staticFileContent != null) {
+            return this.#staticFileContent;
+        }
+
+        const staticFilePath = config.get('staticError.disasterFileContentPath');
+        try {
+            if (staticFilePath != null) {
+                this.#staticFileContent = readFileSync(staticFilePath).toString();
```

Comment:
why sync ?
Maybe it makes sence serve it from memory ?

Reply:
that is an idea. It is put into memory on first read

Reply:
thank you

Reply:
you are welcome, can you approve PR?)